Sources/MapboxMaps/Foundation/Extensions/CoreGraphics.swift

```diff
@@ -24,6 +24,18 @@ extension CGPoint {
         return CGPoint(x: origin.x + fraction * (destination.x - origin.x),
                        y: origin.y + fraction * (destination.y - origin.y))
     }
+
+    /// Returns a new 'CGPoint' adjusted by the input offsets
+    /// - Parameters:
+    ///   - xOffset: The offset in the x direction
+    ///   - yOffset: The offset in the y direction
+    /// - Returns: A new 'CGPoint' with the offets applied
+    internal func shifted(xOffset: CGFloat, yOffset: CGFloat) -> Self {
```

Comment:
```suggestion
    internal func shiftedBy(xOffset: CGFloat, yOffset: CGFloat) -> Self {
```

See https://swift.org/documentation/api-design-guidelines/#argument-labels

<img width="745" alt="Screen Shot 2021-07-09 at 3 00 55 PM" src="https://user-images.githubusercontent.com/205658/125130368-7a9a2180-e0c6-11eb-8da9-fee3136ba635.png">